runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -341,39 +285,9 @@ public void start() {
      */
     public void start(long c) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)